artemis-jms-server/src/main/java/org/apache/activemq/artemis/jms/management/impl/JMSServerControlImpl.java

```diff
@@ -796,20 +796,12 @@ public String listSessionsAsJSON(final String connectionID) throws Exception {
 
       clearIO();
 
-      JsonArrayBuilder array = Json.createArrayBuilder();
       try {
-         List<ServerSession> sessions = server.getActiveMQServer().getSessions(connectionID);
-         for (ServerSession sess : sessions) {
-            JsonObjectBuilder obj = Json.createObjectBuilder()
-                  .add("sessionID", sess.getName())
-                  .add("creationTime", sess.getCreationTime());
-            array.add(obj);
-         }
+         return server.listSessionsAsJSON(connectionID);
       }
       finally {
          blockOnIO();
       }
-      return array.toString();
```

Comment:
Shoot, just noticed this error.  As  soon as this PR is in I'll fix the other place.